src/v0/destinations/marketo_static_list/transform.js

```diff
@@ -107,6 +142,7 @@ const processRouterDest = async (inputs, reqMetadata) => {
   // Token needs to be generated for marketo which will be done on input level.
   // If destination information is not present Error should be thrown
   let token;
+  let respList;
   try {
     token = await getAuthToken(formatConfig(inputs[0].destination));
     if (!token) {
```

Comment:
Rather than generating a JSON for scneario where aut token could not be generated. Throw a Abortable error, this will automatically add necessary stat-tags

Reply:
We need to test this section once we can simply code here